### DISCUSSION TITLE
How can I add multiple affiliations?

### DISCUSSION THREAD

**Discussion by jessicaguo** [quarto-dev/quarto-cli#7014] [Q&A]
### Description

Is it possible to have numbered affiliations show up as super scripts in the author list, and then a section below that lists the number and the full affiliation? I was able to do this previously in Rmarkdown, looking for an example of the equivalent here.

**Comment by mcanouil** [comment]
Likely you'll have to define a template (possibly as an extension) to have this.
Take a look at <https://quarto.org/docs/extensions/listing-journals.html>.
For example <https://github.com/quarto-journals/elsevier>.
